chat-widget/src/components/livechatwidget/common/createAdapter.ts

```diff
@@ -29,6 +29,7 @@ export const createAdapter = async (chatSDK: any) => {
     //so far, there is no need to convert to the shim adapter when using visual tests
     if(chatSDK.isMockModeOn !== true){
```

Comment:
Can you please fix this spacing issue after "if"?

Reply:
done